sandbox/example/package-lock.json

```diff
@@ -1,15 +1,3 @@
 {
-  "name": "@loopback/sandbox-example",
-  "version": "1.0.4",
-  "lockfileVersion": 1,
-  "requires": true,
-  "dependencies": {
-    "@loopback/tslint-config": {
-      "version": "2.0.2",
-      "dev": true,
-      "requires": {
-        "tslint-consistent-codestyle": "^1.14.1"
-      }
-    }
-  }
+  "lockfileVersion": 1
```

Comment:
Unfortunately, lerna does not create package-lock files when a package does not have any external dependencies - see https://github.com/lerna/lerna/issues/2015.

I ran `npm init && npm install` in an empty project to see how a lock file looks for an empty project and used the same content for our sandbox example.

Reply:
Sounds good. Also, would disabling `package-lock` for this sandbox example be a good alternative?

Reply:
Disabling package-lock would work until we add an external dependency, at which point we will almost certainly forget to enable it.